common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -38,7 +39,12 @@ def initialize(source:, dependencies:, files:, credentials:,
       end
 
       def pr_name
-        pr_name = pr_name_prefixer.pr_name_prefix
+        begin
+          pr_name = pr_name_prefixer.pr_name_prefix
+        rescue StandardError => e
+          Dependabot.logger.error("Error while generating PR name: #{e.message}")
+          pr_name = ""
+        end
         pr_name += library? ? library_pr_name : application_pr_name
```

Comment:
Just for the understanding what is the library here?

Reply:
Dependabot differentiates between library and application in a few places. For NPM, if you publish to the registry it's considered a library, otherwise it's an application. I actually didn't look at how this code changes depending on the type, but in another place it decides whether to [increase the version vs widen the range](https://github.com/dependabot/dependabot-core/blob/6fdb1e09bfc8d4f61d2f66ae6eb684f7edf4baa4/npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker.rb#L110).